### PR TITLE
N22 pGLOWrpg: text

### PR DESCRIPTION
**Pull request by roalyr**

### CODE REVIEW

**Comment by roalyr** [comment]
#640

File: content/news/022/index.md [hunk 1]
```diff
@@ -101,6 +101,29 @@ May's full weekly devlogs: "This Week In Veloren...":
...
+a long-term project in development, which aims to be a narrative text-based game
+with maximum portability and accessibility.
+
+Recent updates include:
+
+- Implemented dev features test arena.
+- Implemented entity system draft.
+- Sanitized coordinate systems everywhere (ooof!)
+- Much refactoring.
+- Resumed river generation developmeng.
```
Comment (17cupsofcoffee) [inline content/news/022/index.md:119]:
```suggestion
- Resumed river generation development.
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, just one spelling error:

**Comment by AngelOnFira** [comment]
:rocket:

**Pull request merged**